api/src/db.rs

```diff
@@ -18,7 +19,7 @@ pub struct DeviceWithToken {
 
 pub struct CommandsDB {
     id: i32,
-    cmd: Value,
+    cmd: Json<SafeCommandTx>,
```

Comment:
Are you sure this will work? for some backwards compatible thing, idk maybe worth double check